Source/WebInspectorUI/UserInterface/Base/Utilities.js

```diff
@@ -1866,6 +1866,24 @@ function insertObjectIntoSortedArray(object, array, comparator)
     array.splice(insertionIndexForObjectInListSortedByFunction(object, array, comparator), 0, object);
 }
 
+async function retryUntilTruthy(fn, {delay, retries} = {})
```

Comment:
> this should have tests

Added. 

> NIT: id rename this just `retryUntil` as the "Truthy" part is implicit/redundant. also maybe a name like `callUntil` is more apt since it's possible it may only be called once and therefore not ever "retried"

Went with `retryUntil`. I tried `callUntil` but I felt it didn't communicate the intent as clearly a "retry" when read in context.

> Style: please use something more descriptive than `fn`

Used "predicate"